types/react-toastr/index.d.ts

```diff
@@ -10,10 +10,10 @@ export class ToastContainer extends Component<{
     toastMessageFactory: any;
     className?: string;
 }> {
-    error: (message: string, title: string, optionsOverride?: {}) => void;
-    info: (message: string, title: string, optionsOverride?: {}) => void;
-    success: (message: string, title: string, optionsOverride?: {}) => void;
-    warning: (message: string, title: string, optionsOverride?: {}) => void;
+    error: (message: string | React.ReactNode, title: string | React.ReactNode, optionsOverride?: {}) => void;
```

Comment:
ReactNode includes string, so you only need so specify ReactNode.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L148